examples/readonly.py

```diff
@@ -20,11 +20,15 @@
     print(res)
     time.sleep(0.5)
 
-    since = int(time.time()*1000)-24*60*60*1000
+    since = int(time.time()*1000)-24*60*59*1000
```

Comment:
why this change?

Reply:
`since` can't be older than 24 hours ago.
With `since` equal to exactly 24 hours ago, we get an error, because by the time the server receives the request, it already is > 24h ago.